components/controller.ts

```diff
@@ -593,7 +593,7 @@ export class Controller {
     }
 
     private getGroupContract(json: MissionManifest) {
-        if (json.Metadata.Type === "escalation") {
+        if (["escalation", "arcade"].includes(json.Metadata.Type)) {
```

Comment:
This array comes back a few times and can therefore be considered magic values. Can we move that to a constant somewhere?

Reply:
Sure, this has been done.